server/mqtt.go

```diff
@@ -1947,6 +2045,66 @@ func (as *mqttAccountSessionManager) notifyRetainedMsgDeleted(subject string, se
 	}
 }
 
+func (as *mqttAccountSessionManager) transferUniqueSessStreamsToMuxed(log *Server) {
+	// Set retry to true, will be set to false on success.
+	retry := true
+	defer func() {
+		if retry {
+			next := mqttDefaultTransferRetry
+			log.Warnf("Failed to transfer all MQTT session streams, will try again in %v", next)
+			time.AfterFunc(next, func() { as.transferUniqueSessStreamsToMuxed(log) })
```

Comment:
@derekcollison I wonder if I should cap the retry to once or twice. My concern is that if simply getting the list of streams (say there are thousands of them) fails (line 2060 below), then server would go in this loop of trying to do this, etc..

Reply:
On the list of streams? Should be fine, I would keep trying but maybe space things out in case we are blocked as a cluster etc.

Reply:
You mean use a backoff here? Save the next time and double or something (with a cap)?

Reply:
Just space them out in some form on error. No need for backoff IMO.

Reply:
But they are, every 5 seconds (next := mqtDefaultTransferRetry). I can make this constant 10s instead?